src/melee/ft/chara/ftGameWatch/ftGameWatch_SpecialN.c

```diff
@@ -25,13 +21,117 @@ f32 ftGameWatch_SpecialN_SetFloatOrder3(void)
     return 0.0f;
 }
 
-extern HSD_GObj* func_802C837C(HSD_GObj*, Vec3*, s32, u32, f32);
+#ifdef MUST_MATCH
```

Comment:
`#if`/`#else` order swapped. In general I think it reads better if the matching implementation appears first.